docs/ai/quickstarts/build-mcp-server-publish-nuget.md

```diff
@@ -0,0 +1,219 @@
+---
+title: Quickstart - Create a minimal MCP Server using .NET and publish to NuGet
+description: Learn to create and connect to a minimal MCP server using .NET and publish it to NuGet.
+ms.date: 06/30/2025
+ms.topic: quickstart
+ms.custom: devx-track-dotnet, devx-track-dotnet-ai
+author: jondouglas
+ms.author: jondouglas
+---
+
+# Create, connect and publish a minimal MCP server using .NET to NuGet
+
+In this quickstart, you create a minimal Model Context Protocol (MCP) server using the [C# SDK for MCP](https://github.com/modelcontextprotocol/csharp-sdk), connect to it using GitHub Copilot, and publish it to NuGet. MCP servers are services that expose capabilities to clients through the Model Context Protocol (MCP).
+
+## Prerequisites
+
+- [.NET 10.0 SDK or higher](https://dotnet.microsoft.com/download)
```

Comment:
I think we should mentioned .NET 10 Preview 6 here for now. If they have an earlier preview it won't work. We can clean i up later if needed.

Reply:
Ill change to the download/dotnet page as it actually has 10.0 listed, but this should be fine to not have to mention preview as it means we wont have to change the doc later.